playbooks/roles/streisand-client/vars/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Path to the gateway password on the Ansible host
+streisand_gateway_password_localpath: "../{{ streisand_local_directory }}/gateway-password.txt"
+
+# Path to the gateway password file on the streisand client
+gateway_password_file: "/root/gateway-password.txt"
+# URL for accessing the streisand server's HTTP gateway page
+gateway_test_url: "https://{{ streisand_ip }}"
+# User for accessing the streisand server HTTP gateway page
+gateway_test_user: "streisand"
+
+# The directory to copy test scripts to
+test_script_dir: "/usr/local/bin/"
+
+# Test scripts to install & run
+test_scripts:
+  - { template: 'streisand-gateway-test.sh.j2', cmd: 'streisand-gateway-test' }
```

Comment:
I'm happy to reformat to align columns. I think the second approach will be more duplication since the current `test_scripts` var entries are used in a couple places.